REDCapUITweaker.php

```diff
@@ -587,6 +628,39 @@ function redcap_every_page_top( $project_id = null )
 					}
 				}
 
+
+				// If the @SQLCHECKBOX action tag is enabled, remove the SQL which provides the
```

Comment:
Does this or can this provide a warning that is visible before a project is run?  If someone upgrades their version of the UI Tweaker on an existing project, how does deprecation message work in practice - will an existing project still run without the warning, but if they open it in design mode, the warning appears?

Reply:
Yes, a warning message will be displayed in the designer when opening an instrument with fields using the old `@SQLCHECKBOX` implementation (i.e. where the action tag has been placed on a checkbox field). The data entry process will still function as before without any warnings.